src/TypeScript/EslintBridgeClient/EslintBridgeClient.cs

```diff
@@ -40,21 +40,22 @@ internal interface IEslintBridgeClient : IDisposable
     /// Matching Java implementation: https://github.com/SonarSource/SonarJS/blob/0dda9105bab520569708e230f4d2dffdca3cec74/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/eslint/JavaScriptEslintBasedSensor.java#L51
     /// Eslint-bridge methods: https://github.com/SonarSource/SonarJS/blob/0dda9105bab520569708e230f4d2dffdca3cec74/eslint-bridge/src/server.ts
     /// </summary>
-    [Export(typeof(IEslintBridgeClient))]
-    [PartCreationPolicy(CreationPolicy.Shared)]
     internal sealed class EslintBridgeClient : IEslintBridgeClient
     {
+        private readonly Uri baseServerUri;
         private readonly IEslintBridgeHttpWrapper httpWrapper;
         private readonly IAnalysisConfiguration analysisConfiguration;
 
-        [ImportingConstructor]
-        public EslintBridgeClient(IEslintBridgeHttpWrapper httpWrapper)
-            : this(httpWrapper, new AnalysisConfiguration())
+        public EslintBridgeClient(Uri baseServerUri, ILogger logger)
```

Comment:
This is now a utility class, similar to the HttpWrapper.
Since the process and can die and start on a new port, it receives the server uri in the ctor, so you need to create a new one if the port has changed. This is done by JavaScriptAnalyzer.